dialogue-core/src/main/java/com/palantir/dialogue/core/RetryingChannel.java

```diff
@@ -98,6 +99,7 @@
     private final class RetryingCallback {
         private final Endpoint endpoint;
         private final Request request;
+        private final DetachedSpan span = DetachedSpan.start("Dialogue-RetryingChannel");
```

Comment:
ooh nice so this actually won't appear if a request succeeds first time?

Reply:
Yep :-)

Reply:
The successful request tracing test is not impacted by this change